### ISSUE TITLE
Disable autosave toggle does not meet color contrast threshold

### ISSUE THREAD

**Issue by VirginiaBalseiro** [linkeddata/dokieli#505]
The color of the text ‘Autosave’ (#777) doesn’t meet minimum color contrast ratio thresholds.

See: https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum

**Comment by VirginiaBalseiro** [comment]
This is a false positive from the fact that the background of the label is #777 and the color of the text in the label is black, but this does not actually affect the user because the elements and pseudoelements in the labeled are absolutely positioned to style the toggle in that particular shape, so in the end the background of the black text is actually the white background in the user menu.
